app/jobs/cron_job/crm_sync.rb

```diff
@@ -0,0 +1,47 @@
+class CronJob::CrmSync < ApplicationJob
+  NOTION_DATABASE_ID = "81a35c7b2490464590a408bbbb78ca2e".freeze
+
+  def perform
+    return unless ENV["NOTION_API_SECRET"]
+
+    filter = {
+      and: [
+        {
+          property: "COMPTE PROD",
+          url: {
+            contains: instance_hostname,
+          },
+        },
+      ],
+    }
+
+    client.database_query(database_id: NOTION_DATABASE_ID, filter:) do |page|
+      page.results.each do |notion_page|
+        rdv_count = Rdv.where(organisation: organisations_ids(notion_page.properties["COMPTE PROD"].url)).count
+        client.update_page(page_id: notion_page.id, properties: { "NOMBRE DE RDV" => rdv_count })
+      end
+    end
+  end
+
+  private
+
+  def organisations_ids(account_url)
+    if account_url.match('territories/(\d+)')
+      territory_id = account_url.match('territories/(\d+)')[1]
+      territory = Territory.find(territory_id)
```

Comment:
```suggestion
      territory = Territory.find($1)
```

si tu es un hacker tu peux utiliser la variable `$1` qui contient le match entre parenthèse exécuté le plus récemment cf https://ruby-doc.org/docs/ruby-doc-bundle/Manual/man-1.4/variable.html#number

Reply:
Hyper intéressant
En revanche pas vraiment fan parce que ça rend le code pas très clair